src/main/java/antlr/treeFormatParser/generated/TreeFormat.interp

```diff
@@ -0,0 +1,30 @@
+token literal names:
```

Comment:
Ignore generate files in Antrl.
![](https://media.giphy.com/media/XZxE5KQc0fkSthaMxh/giphy.gif)

Reply:
Well, this has to be addressed in the future